products/bash/deploy-og-sub.sh

```diff
@@ -29,9 +30,13 @@ function usage() {
 }
 
 namespace="cp4i"
+DEPLOY_DEMOS=false
 
-while getopts "n:p" opt; do
+while getopts "n:d" opt; do
   case ${opt} in
+  p)
```

Comment:
Does this need to be `d` to match the arg?

Reply:
Ooops, yes!